### ISSUE TITLE
Dependency on CanFestival Not Linked

### ISSUE THREAD

**Issue by DLu** [swri-robotics/euler#28]
```
vetex_driver/src/vetexcanopen.cpp:36:25: fatal error: canfestival.h: No such file or directory compilation terminated.
```

May be related to #27.

**Comment by DLu** [comment]
Ok, just found the relevant part of the Readme. Still, the extra step makes compilation trickier. Would be better if it was integrated as part of the make process.

**Comment by shaun-edwards** [comment]
I completely agree.  I don't have any experience with 3rd party/non-catkin dependencies, so I'm not sure what the solution is.  Sorry for the confusion.
